src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -616,32 +617,57 @@ public Task GetPeriodicBackupStatus()
             await DatabaseConfigurations((_, databaseName, etlConfiguration) => ServerStore.UpdateEtl(_, databaseName, id, etlConfiguration, etlType), "etl-update");
         }
 
-        [RavenAction("/admin/console", "POST", "/admin/console?name={databaseName:string}")]
+        [RavenAction("/admin/console", "POST", "/admin/console?database={databaseName:string}")]
         public async Task AdminConsole()
         {
-            var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
-            if (ServerStore.DatabasesLandlord.DatabasesCache.TryGetValue(name, out var database) && database.Status == TaskStatus.RanToCompletion)
+            var name = GetStringValuesQueryString("database", false);
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             {
-                using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+                var content = await context.ReadForMemoryAsync(RequestBodyStream(), "read-admin-script");
+                if (content.TryGet(nameof(AdminJsScript), out BlittableJsonReaderObject adminJsBlittable) == false)
+                {
+                    throw new InvalidDataException("AdminJsScript was not found.");
+                }
+
+                var adminJsScript = JsonDeserializationCluster.AdminJsScript(adminJsBlittable);
+                DynamicJsonValue result = null;
+
+                if (name.Equals(default(StringValues)) == false)
```

Comment:
We can't take the absence of name only as a hint to run at server level, need an explicit request by user with another arg.